src/AssetLoader.php

```diff
@@ -226,6 +233,13 @@ public function withCssDefaultOptions(array $cssDefaultOptions): self
         return $new;
     }
 
+    public function withCssDefaultPosition(?int $position): self
```

Comment:
Probably need to specify in docblock what `int ' stands for, what possible values are.

Reply:
Done.